iotedgedev/solution.py

```diff
@@ -33,7 +33,7 @@ def create(self, name, module, template):
 
         os.rename(os.path.join(name, ".env.tmp"), os.path.join(name, ".env"))
 
-        mod_cmd = "iotedgedev addmodule {0} --template {1}".format(module, template)
+        mod_cmd = "iotedgedev solution add {0} --template {1}".format(module, template)
```

Comment:
Should this be addmodule or just add? This might be misinterpreted as "add a solution"

Reply:
This is a tradeoff between convenience and expressiveness. Adding modules should be a frequently used command (more than the create solution command), so making it shorter can save keystrokes. On the other hand, given the existence of the `create` command and the help messages, I suppose the misinterpretation to be unlikely.